docs/kore-syntax.md

```diff
@@ -0,0 +1,143 @@
+Syntax of the KORE language
+===========================
+
+This documents describes the syntax of KORE, as admitted by the Haskell Backend.
+
+It is meant to be an easier to access source than the
+[Semantics of K](docs/semantics-of-k.pdf)
+
+Any change in the KORE Parser infrastructure should be reflected here.
+
+Lexical Syntax
+--------------
+
+### Comments
+
+- C-style comments (non-nested)
+
+### Strings
+
+```
+StringLiteral ::= "\"" {Char}* "\""
+
+Char ::= EscapeChar || AsciiChar || PrintableChar
+
+AsciiChar ::= first 128 ASCII characters
```

Comment:
I just stated what our parser accepts. maybe that needs to change, too.

Reply:
Does our parser really accept """ as a valid string? I thought it will take the first two quotation marks as being the string and produce an error at the third.